docs/plugins/filters/memcached.asciidoc

```diff
@@ -5,9 +5,9 @@
 ///////////////////////////////////////////
 START - GENERATED VARIABLES, DO NOT EDIT!
 ///////////////////////////////////////////
-:version: v1.0.1
-:release_date: 2019-05-31
-:changelog_url: https://github.com/logstash-plugins/logstash-filter-memcached/blob/v1.0.1/CHANGELOG.md
+:version: v1.0.0
+:release_date: 2019-02-04
+:changelog_url: https://github.com/logstash-plugins/logstash-filter-memcached/blob/v1.0.0/CHANGELOG.md
```

Comment:
Regression caused by lockfile entry.  I make a commit to fix formatting that would crash the doc build.